config/initializers/sidekiq.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'scholarsphere/redis_config'
+
+redis_config = Scholarsphere::RedisConfig.new.call
```

Comment:
@whereismyjetpack see  my early comments, but I think just `Scholarsphere::RedisConfig.new` here and then redis_config.config or redis_config.to_hash or something...